lib/rubocop/cop/style/trailing_body_on_method_definition.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for trailing code after the method definition.
+      #
+      # @example
+      #   # bad
+      #   def some_method; do_stuff
+      #   end
+      #
+      #   # good
+      #   def some_method
+      #     do_stuff
+      #   end
+      #
+      class TrailingBodyOnMethodDefinition < Cop
+        include AutocorrectAlignment
+
+        MSG = 'Put the method definition body on its own line.'.freeze
```

Comment:
Place the first line of a multi-line method definition's body on its own line.

Reply:
Thanks! That's a very clear message.